.github/workflows/reusable-build-on-ubuntu.yml

```diff
@@ -109,3 +109,7 @@ jobs:
           name: codecov-wasmedge
           fail_ci_if_error: true
           verbose: true
+      - name: Run post-build check (check_symbols.sh)
+        run: |
+          chmod +x ./scripts/check_symbols.sh
+          ./scripts/check_symbols.sh
```

Comment:
I'd prefer to keep this step the same across different platforms if possible. If `bash ./scripts/check_symbols.sh` works on Windows without `chmod +x`, can we use the same one line here?